src/registrar/fixtures_users.py

```diff
@@ -86,6 +86,12 @@ class UserFixture:
             "first_name": "Kristina",
             "last_name": "Yin",
         },
+        {
+            "username": "ac49d7c1-368a-4e6b-8f1d-60250e20a16f",
+            "first_name": "Vicky",
+            "last_name": "Chin",
+            "email": "szu.chin@associates.cisa.dhs.gov",
```

Comment:
Just making sure: does she want to use this email for Admin?

Reply:
+1 to Zanders comment, but it's non-blocking.

If Vicky doesn't wan't to keep swapping between two email tabs, we may want to keep up with the some_email for Admin and then some_email+1 for Analyst pattern